lib/widgets/misc/text/body.dart

```diff
@@ -4,18 +4,19 @@ import 'package:flutter/material.dart';
 class Body extends StatelessWidget {
   final String text;
   final TextAlign align;
+  final double size;
 
   ///Returns a custom [Text] widget for accessing body1 theme
   ///and has an optional alignment property.
-  Body(this.text, {this.align = TextAlign.center});
+  Body(this.text, {this.align = TextAlign.center, this.size = 0});
 
   @override
   Widget build(BuildContext context) {
     return Text(
       text,
       overflow: TextOverflow.fade,
       style: TextStyle(
-        fontSize: Screen.width(context) <= 350 ? 16 : 20,
+        fontSize: size == 0 ? Screen.width(context) <= 350 ? 16 : 20 : size,
```

Comment:
then this can probably just be: `fontSize: size ?? Screen.width(context) <= 350 ? 16 : 20`

And when you use this widget you can leave not pass through a size to use the default sizes

Reply:
this might not work as we intend at this stage I will make a low priority task for it